pkgs/tools/misc/coreutils/default.nix

```diff
@@ -33,6 +33,9 @@ stdenv.mkDerivation (rec {
     ./fix-chmod-exit-code.patch
     # Workaround for https://debbugs.gnu.org/cgi/bugreport.cgi?bug=51433
     ./disable-seek-hole.patch
+    # Workaround for https://debbugs.gnu.org/cgi/bugreport.cgi?bug=52330
+    # This patch can be dropped after the next coreutils release + upgrade
```

Comment:
Not a big deal (I'm happy to merge as is), I might have added something detailing the current version or the current date, just so people know when to remove this.
Like I said, not a big deal, happy to merge as is.

Reply:
I think you are right, I reworded this accordingly :) ty